sentry-samples/sentry-samples-console/src/main/java/io/sentry/samples/console/Main.java

```diff
@@ -0,0 +1,132 @@
+package io.sentry.samples.console;
+
+import io.sentry.core.Breadcrumb;
+import io.sentry.core.EventProcessor;
+import io.sentry.core.Sentry;
+import io.sentry.core.SentryEvent;
+import io.sentry.core.SentryLevel;
+import io.sentry.core.protocol.Message;
+import io.sentry.core.protocol.User;
+import java.util.Collections;
+
+public class Main {
+
+  public static void main(String[] args) {
+    Sentry.init(
+        options -> {
+          // NOTE: Replace the test DSN below with YOUR OWN DSN to see the events from this app in
+          // your Sentry project/dashboard
+          options.setDsn("https://f7f320d5c3a54709be7b28e0f2ca7081@sentry.io/1808954");
```

Comment:
Lets have this one by default (used in the other sample in the repo)
```suggestion
          options.setDsn("https://f7f320d5c3a54709be7b28e0f2ca7081@sentry.io/1808954");
```

Reply:
Oh nevermind, it's the right DSN :)

Reply:
They look the same to me..?

Reply:
I need to get my brain checked, sorry